pytripgui/main.py

```diff
@@ -23,29 +23,27 @@
 
 import wx
 import wx.lib.dialogs
-from wx.xrc import *
+from wx.xrc import XRCCTRL, XRCID, XmlResource
 
-from pytrip.error import *
+from pytrip.error import InputError
 
-from pytripgui.leftmenu import *
-from pytripgui.settings import *
-from pytripgui.plugin import *
+from pytripgui.leftmenu import LeftMenuTree
+from pytripgui.settings import SettingsManager
+from pytripgui.plugin import PluginManager
 
-from pytripgui.panels.plotpanel import *
-from pytripgui.panels.dvh import *
+from pytripgui.panels.plotpanel import PlotPanel
+from pytripgui.panels.dvh import DVHPanel, LVHPanel
 
-from pytripgui.tripexecparser import *
-from pytripgui.data import *
+from pytripgui.data import PytripData
+from pytripgui.util import get_resource_path
 from pytripgui import util
 
 if getattr(sys, 'frozen', False):
-    from wx.lib.pubsub import setuparg1
```

Comment:
Unfortunately it is needed there in order to have Windows version working :/
Could you please revert it ?

You can use `#noqa` to disable flake8 checks on some lines, see https://pypi.python.org/pypi/flake8

- files that contain this line are skipped::
    ```# flake8: noqa```
- lines that contain a ``# noqa`` comment at the end will not issue warnings.
- you can ignore specific errors on a line with ``# noqa: <error>``, e.g.,
  ``# noqa: E234``